pyproject.toml

```diff
@@ -26,10 +27,9 @@ classifiers = [
    "Programming Language :: Python :: 3.10",
```

Comment:
This looks like it should be removed. We are leaving only the `Python :: 3 :: Only` classifier in other projects as the range of supported versions is already specified in `requires-python`, so we avoid the extra maintenance cost of keeping classifiers up to date.

Reply:
Yeah, I would keep it as it is zero maintenance effort and it works in Python 3.